### PR TITLE
Fix some bugs in nightcore command

### PR DESCRIPTION
**Pull request by NathanTW0219**

### CODE REVIEW

**Review by Gary50613 — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "express-ws": "^5.0.2",
     "ffmpeg-static": "^5.0.0",
     "helmet": "^5.1.0",
+    "heroku": "^7.60.2",
```
Comment (wolf-yuan-6115) [inline package.json:43]: Why adding heroku deps?

Comment (Gary50613) [reply package.json:43]: maybe he tried to use the cli but forgot to use `-g` while installing

**Review by Gary50613 — APPROVED** [review]
LGTM

**Pull request merged**